homeassistant/components/google_generative_ai_conversation/manifest.json

```diff
@@ -8,5 +8,5 @@
   "documentation": "https://www.home-assistant.io/integrations/google_generative_ai_conversation",
   "integration_type": "service",
   "iot_class": "cloud_polling",
-  "requirements": ["google-generativeai==0.8.2"]
+  "requirements": ["google-genai==1.1.0"]
```

Comment:
@tronikos what's the policy with the versions in HA, do they need to be pinned? In theory all 1.x.x versions are backward compatible. So we could just pin it to 1.x.x

Reply:
Yes they need to be pinned.